roles/contiv_network/tasks/aci_tasks.yml

```diff
@@ -23,6 +23,7 @@
   service: name=aci-gw state=started
   run_once: true
 
+# NOTE: Use Contivctl command when it is available
 - name: set aci mode
-  shell: contivctl global set -nwinfra aci
```

Comment:
@erikh : I was looking at the contivctl implementation and looks like it just points to netctl commands. Is that right or is it packaged with a separate netctl binary? So, this is what I see... https://gist.github.com/DivyaVavili/66e80e9e16f9330bce6b